README.rst

```diff
@@ -12,22 +12,23 @@ Then install all the requirements using
 `pip3 install -r requirements.txt`
 
 Running test suite
-#################
+##################
 
 To run test suite
 `python3 -m pytest`
 
 Usage of API to fetch HTTP/S and FTP URLs
 #########################################
-```
-from fetchcode import fetch
-url = 'A Http or FTP URL'
-location = 'Location of file'
-# This returns a response object which has attributes
-# 'content_type' content type of the file
-# 'location' the absolute location of the files that was fetched
-# 'scheme' scheme of the URL
-# 'size' size of the retrieved content in bytes
-# 'url' fetched URL
-resp = fetch(url = url)
-```
+
+.. code-block:: python
+
+    from fetchcode import fetch
+    url = 'A Http or FTP URL'
+    location = 'Location of file'
```

Comment:
Line number 27 here :)

Reply:
Fixed

Reply:
Thanks, can you squash your commits :)

Reply:
Done, is now is it ok

Reply:
Does it have good quality to merge the code